main/src/library/Time/Instant.flix

```diff
@@ -36,26 +36,27 @@ namespace Time {
         ///
         @Experimental
         pub def before(x: Time.Instant, y: Time.Instant): Bool =
-            import java.time.Instant.isBefore(##java.time.Instant);
+            import java.time.Instant.isBefore(##java.time.Instant): Bool & Pure;
             let Time/Instant.Instant(t1) = x;
             let Time/Instant.Instant(t2) = y;
-            (t1 `isBefore` t2) as & Pure
+            (t1 `isBefore` t2)
 
         ///
         /// Returns `true` if `x` is after `y`.
         ///
         @Experimental
         pub def after(x: Time.Instant, y: Time.Instant): Bool =
-            import java.time.Instant.isAfter(##java.time.Instant);
+            import java.time.Instant.isAfter(##java.time.Instant): Bool & Pure;
             let Time/Instant.Instant(t1) = x;
             let Time/Instant.Instant(t2) = y;
-            (t1 `isAfter` t2) as & Pure
+            (t1 `isAfter` t2)
 
         ///
         /// Returns the given instant `x` plus `d` number of nano seconds.
         ///
         @Experimental
         pub def plus(x: Time.Instant, d: Int64): Time.Instant =
+            // TODO: refactor import.
             import java.time.Instant.plusNanos(Int64);
             let Time/Instant.Instant(t) = x;
             Time/Instant.Instant(t `plusNanos` d) as & Pure
```

Comment:
I'm not sure what to do with this one.
If I write
```
import java.time.Instant.plusNanos(Int64): Time.Instant & Pure;
```
It says `>> Unable to unify the types: 'Instant' and 'Time.Instant'.`.
And if I write
```
import java.time.Instant.minusNanos(Int64): Instant & Pure;
```
It says `>> Undefined type 'Instant'.`.

Any recommendations?

Reply:
I think you need to put the full Java type as the signature.

`import java.time.Instant.plusNanos(Int64): ##java.time.Instant & Pure;`

The Java `Instant` and the Flix `Instant` are different types.